### PR TITLE
Installation from source - setting python interpreter version

### PR DESCRIPTION
**Pull request by sploitable**
If you have python 3.7 installed and you would like to force 3.6 when installing from source.

Tested with MacOS High Sierra.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -324,12 +324,8 @@ While we try not to break master, there are no guarantees on anything. ::
...
     git clone https://github.com/borgbackup/borg.git
 
-    # set python interpreter version e.g. 3.6.0
-    pyenv install 3.6.0
-    pyenv global 3.6.0
-    pyenv local 3.6.0
-
-    virtualenv --python=${pyenv which python} borg-env
+    # create a virtual environment
+    virtualenv --python=${which python3} borg-env
```
Comment (RonnyPfannschmidt) [inline docs/installation.rst]: for more modern python3, one should use python3 -m venv

Comment (ThomasWaldmann) [reply docs/installation.rst]: @RonnyPfannschmidt ok, but guess this is something for a global docs / scripts review and then change it everywhere, in a separate PR.

**Comment by ThomasWaldmann** [comment]
thanks!

**Comment by ThomasWaldmann** [comment]
if we want to have this in the docs rather soon, a pull request with a backport to 1.1-maint branch would be nice - can you do it?

**Pull request merged**